CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+Release v2.3.0 (2024-01-12)
+===========================
+### Modified
+1. Update permission check logic to support latest namespace roles format ```{studio}+```
+2. this version need following config if in multi-tenant:
+   * basic service base url: BasicBaseURL
```

Comment:
Does it have a default value? Let's first remove this item configurable and directly use the default one to simply the configuration since they will be always in the same cluster.